libledger/Ledger.cpp

```diff
@@ -565,15 +574,24 @@ bool Ledger::initBlockChain(GenesisBlockParam& _genesisParam)
     return true;
 }
 
+bool Ledger::setUpWithRotatingPBFT()

```

Comment:
I suggest this is uncessary, maybe more simple

Reply:
This function has have been called too many times.